nucypher/policy/maps.py

```diff
@@ -36,18 +35,19 @@
 from nucypher.crypto.signing import SignatureStamp
 from nucypher.network.middleware import RestMiddleware
 from umbral.keys import UmbralPublicKey
+from umbral.kfrags import KFrag
 from umbral.signing import Signature
 
 
-class TreasureMap:
+class TreasureMapSplitter(BrandingMixin, VersioningMixin, BytestringKwargifier):
+    pass
+
 
-    VERSION_NUMBER = 2  # Increment when serialization format changes.
+class TreasureMap:
+    VERSION_NUMBER = 1  # Increment when serialization format changes.
 
-    _DELIMITER = b':'
-    _PREFIX = b'TM' + _DELIMITER
-    _VERSION = int(VERSION_NUMBER).to_bytes(1, 'big')
-    _HEADER = _PREFIX + _VERSION
-    _HEADER_SIZE = len(_HEADER)
+    _BRAND = b'TM'
+    _VERSION = int(VERSION_NUMBER).to_bytes(2, 'big')
```

Comment:
Maybe it's better to have `VersioningMixin.HEADER_LENGTH` instead of `2`. And perhaps some control for the `_BRAND` length as well.

Reply:
Agreed.